R/derive_flag_select_msrc.R

```diff
@@ -0,0 +1,267 @@
+#' Merge an Existence Flag From Multiple Sources
+#'
+#' @description Adds a flag variable to the input dataset which indicates if
```

Comment:
What do you think of adding an high-level example at the top similar to to the example in the code?  Just to give folks a little more information on how it might be used before they get to the arguments .

Reply:
Added